release/default.nix

```diff
@@ -322,6 +326,30 @@ let
     in
     {
 
+      fc-options = let
```

Comment:
Possibly add a descriptive comment about the purpose of this image. According to line 392, this is more kind of a dummy/ rpxy and not intended to output an image, but to grab the options from it.

Reply:
I decided to move the image and the generation of the options to the fc-search module to keep this part of the code cleaner and more flexible.

Reply:
I cannot find that module's code in this PR, I assume you're going to provide that separately at a later point?

Reply:
yes, it lives in https://github.com/flyingcircusio/fc-search

since generating the options for each channel is just a matter of about 5 seconds, there is no need to do t on hydra at the cost of less maintainable code